src/com/ean/clarity/Main.java

```diff
@@ -0,0 +1,53 @@
+package com.ean.clarity;
+
+import com.ean.clarity.util.Baugus;
```

Comment:
This package is known for security risks involving serious piracy and possible beatings.